src/components/Person/Person.jsx

```diff
@@ -1 +1,15 @@
-// export const Person = ({ person }) => ();
+import '../../App.scss';
```

Comment:
You need to add scss, cause you already have it in App.jsx

Reply:
Good point, fix this

Reply:
Not fixed